apps/events/forms.py

```diff
@@ -52,12 +54,26 @@ class ScheduledMessageConfigForm(forms.Form):
         min_value=1,
         help_text="Indicates how many times this should go on for. Specify '1' for a one time event",
     )
+    experiment_id = forms.ChoiceField(
+        label="Experiment", help_text="Select the child experiment to process this scheduled message"
```

Comment:
nit: wording (this list will include the 'parent')
```suggestion
        label="Experiment", help_text="Select the experiment to process this scheduled message"
```